tests/test_af1.py

```diff
@@ -96,18 +96,17 @@ def test_site_filters(region):
     assert filter_pass.dtype == bool
 
 
-@pytest.mark.parametrize("chunks", ["auto", "native"])
```

Comment:
No need to cover different values of `chunks` parameter here I think, should already be well covered in tests on Ag3 class.